### ISSUE TITLE
ICE: Assertion 'copied > 0' failed at /ecma-helpers-string.cpp(ecma_is_string_magic_longpath):1543

### ISSUE THREAD

**Issue by egavrin** [Samsung/jerryscript#740]
Version: 17a5dd27bbbadc036d83be0448f9edfa90800147

System:
```
Linux eg 4.2.0-18-generic #22-Ubuntu SMP Fri Nov 6 18:25:50 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```

Output:
```

ICE: Assertion 'copied > 0' failed at /home/eg/dev/jr/jerry-core/ecma/base/ecma-helpers-string.cpp(ecma_is_string_magic_longpath):1543.
Error: ERR_FAILED_INTERNAL_ASSERTION

```
Testcase:

```js
VVVXyieZptiontV֡VVVVVVRVVVXyield;
```

**Comment by renatahodovan** [comment]
This is a duplicate of #380.

**Comment by egavrin** [comment]
Closed as #380 duplicate.
